website/static/js/osfHelpers.js

```diff
@@ -510,6 +510,20 @@ ko.bindingHandlers.listing = {
     }
 };
 
+/* Responsive Affix for side nav */
+function fixAffixWidth() {
+    $('.affix, .affix-top, .affix-bottom').each(function (){
+        var el = $(this);
+        var colsize = el.parent('.affix-parent').width();
+        el.outerWidth(colsize);
+    });
+}
+
+var initializeResponsiveAffix = function (){
+    $(window).resize(function (){ fixAffixWidth(); });
```

Comment:
Use trial-and-error. Indeed, 100ms will definitely be perceptible, so try out lower values.